src/main/scala/higherkindness/rules_scala/common/worker/WorkerMain.scala

```diff
@@ -86,10 +86,18 @@ abstract class WorkerMain[S](stdin: InputStream = System.in, stdout: PrintStream
 
           val requestId = request.getRequestId()
 
+          // "Passing the --worker_verbose flag to Bazel sets the verbosity field to 10, but smaller
+          // or larger values can be used manually for different amounts of output."
+          // For more info: https://bazel.build/remote/creating#work-requests
+          val verbosity = request.getVerbosity()
+          val isVerbose = verbosity >= 10
+
           // If this is a cancel request, we need to cancel a previously sent WorkRequest
           // Arguments and inputs fields on cancel requests "must be empty and should be ignored"
           if (request.getCancel()) {
-            System.err.println(s"Cancellation WorkRequest received for request id: $requestId")
+            if (isVerbose) {
```

Comment:
Is there any way we can hook this into a logging framework so we don't have to manually check the verbosity every time we log?

Even a helper function would be better.

Reply:
I'll add a quick function for this. We could write a logger for it, but not sure it's worth doing that unless we start doing this in more places.

We do some logger things in workers, so if I'm ever working on those again, maybe I'll change things here to  use a logger.

Reply:
Sounds good. Thanks!